.github/workflows/performance-testing.yml

```diff
@@ -18,26 +18,24 @@ jobs:
         with:
           node-version: ${{ vars.NODE_VERSION }}
 
-      - name: Cache pnpm dependencies
-        id: cache-pnpm-dependencies
-        uses: actions/cache@v4.2.3
-        with:
-          path: node_modules
-          key: ${{ runner.os }}-dependencies-${{ hashFiles('**/pnpm-lock.yaml') }}
-          restore-keys: |
-            ${{ runner.os }}-dependencies-
-
-      - name: Install pnpm
-        run: npm install -g pnpm
+      - name: Start container
+        run: make start
 
-      - name: Install dependencies
-        run: pnpm install --frozen-lockfile
-        if: |
-          steps.cache-pnpm-dependencies.outputs.cache-hit != 'true' ||
-          false
+      - name: Install Chromium in dev container
+        run: docker compose exec -T dev apk add --no-cache chromium nss freetype ttf-freefont harfbuzz
```

Comment:
All container dependencies should be installed during the docker build phase and this code should be moved from GitHub actions workflow to the appropriative Dockerfile